.github/workflows/ci.yaml

```diff
@@ -60,28 +60,44 @@ jobs:
         pip install -r tests/requirements-test.txt
         python -m pytest -v -p no:warnings
         deactivate
+  deployment-tests:
+    runs-on: self-hosted
+    strategy:
+      matrix:
+        profile: ["local-code", "latest-release"]
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v2
     - name: Setup Cluster
       run: |
         kind delete cluster 2>/dev/null
         kind create cluster
         kubectl get nodes
     - name: Deploy From Source
+      if: ${{ matrix.profile != 'latest-release' }}
       run: |
         skaffold config set --global local-cluster true
-        skaffold run --default-repo local
+        skaffold run --default-repo local \
+                     --profile ${{ matrix.profile }}
+    - name: Deploy Latest Release
+      if: ${{ matrix.profile == 'latest-release' }}
+      run: |
+        set -x
+        kubectl apply -f extras/jwt
+        kubectl apply -f kubernetes-manifests
     - name: Wait For Pods
       timeout-minutes: 20
       run: |
         set -x
-        kubectl wait --for=condition=ready --timeout=500s pod/accounts-db-0
-        kubectl wait --for=condition=ready --timeout=500s pod/ledger-db-0
         kubectl wait --for=condition=available --timeout=500s deployment/frontend
         kubectl wait --for=condition=available --timeout=500s deployment/contacts
         kubectl wait --for=condition=available --timeout=500s deployment/userservice
         kubectl wait --for=condition=available --timeout=500s deployment/ledgerwriter
         kubectl wait --for=condition=available --timeout=500s deployment/transactionhistory
         kubectl wait --for=condition=available --timeout=500s deployment/balancereader
         kubectl wait --for=condition=available --timeout=500s deployment/loadgenerator
+        kubectl wait --for=condition=ready --timeout=500s pod/accounts-db-0
```

Comment:
qq: were these moved for functional or formatting reasons?

Reply:
Surprisingly, functional. "Deploy Latest Release" fails if this is on top

The db manifests create a statefulset, but you can't wait on that directly, you have to wait on the underlying Pod. But the Pod takes a couple seconds to appear, and if you wait on it before it exists, it will return an error

Reply:
well that sounds like it was fun to debug 😄 thanks for explaining

Reply:
I definitely tried a couple hacky workarounds before realizing I could just move this line to the bottom!